recipes/python-periodictable/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-periodictable" %}
```

Comment:
Yes, the directory should match the package name. It's a bit silly, but we have issues if it doesn't.

xref: https://github.com/conda-forge/staged-recipes/issues/1073